core/vm/contracts_test.go

```diff
@@ -323,6 +326,7 @@ func BenchmarkPrecompiledBLS12381MapG1(b *testing.B)      { benchJson("blsMapG1"
 func BenchmarkPrecompiledBLS12381MapG2(b *testing.B)      { benchJson("blsMapG2", "12", b) }
 
 // Failure tests
+func TestPrecompiledBn256PairingFail(t *testing.T)       { testJsonFail("bn256Pairing", "08", t) }
```

Comment:
better move this line to line 264 because this section is all almost BLS curve testing.